tests/lib/prepare-project.sh

```diff
@@ -68,17 +68,18 @@ fi
 
 # declare the "quiet" wrapper
 . "$TESTSLIB/quiet.sh"
+. "$TESTSLIB/dirs.sh"
```

Comment:
It will be exactly one `spread.yaml`, there will be many lines but still just one place to look :)